test/controllers/api/v2/host_collections_controller_test.rb

```diff
@@ -61,7 +61,7 @@ def test_show
     def test_create
       post :create, params: { :organization_id => @organization, :host_collection => {:name => 'Collection A', :description => 'Collection A, World Cup 2014'} }
 
-      assert_response :success
+      assert_response (201)
```

Comment:
Let's be consistent with our style. I'd like to see `assert_response 201` or `assert_response(201)` (note no space when using parenthesis)

Reply:
rubocop complained about it as well.